ai_summarizer/services.py

```
@@ -0,0 +1,18 @@
+from clinical_refs.models import ClinicalReference
+
+TOPIC_KEYWORDS = {
+    'diabetes': ['diabetes','hba1c','metformin','insulin','sglt2','glp-1'],
+}
+
+def link_references(summary_text, topic_hint='diabetes'):
```

Ruff (0.12.2)

Missing return type annotation for public function `link_references`

(ANN201)